src/test/App.test.js

```diff
@@ -0,0 +1,51 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
- [ ] Some tests are failing, kindly fix them 
![image](https://user-images.githubusercontent.com/62705947/225369192-627ab952-489b-4762-a9db-01fcaf113893.png)

Reply:
- [ ] Please add tests for some more components, currently, you are testing only the app component

Reply:
but the weather content changes based on the date